localization/pose_initializer/src/map_height_fitter/map_height_fitter_node.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Autoware Foundation
```

Comment:
Please make sure the copyright is correct.

Reply:
Done https://github.com/autowarefoundation/autoware.universe/pull/1500/commits/6c825d757eaf512ba59b39b108a32936c74f0348.